contracts/L2/gateway/L2Migrator.sol

```diff
@@ -140,6 +166,58 @@ contract L2Migrator is L2ArbitrumMessenger, IMigrator {
 
     receive() external payable {}
 
+    // Assume that only EOAs are included in the snapshot
+    // Regardless of the caller of this function, the EOA from L1 will be able to access its stake on L2
+    function claimStake(
```

Comment:
At the moment, this function can be called by anyone. But, we should actually only allow this to be called by `_delegator` (i.e. when `_delegator == msg.sender`) because `_newDelegate` is specified by the caller and only `_delegator` should be allowed to to specify a new delegate.

Will push up a fix.

Reply:
Fixed in d78e7a6